CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Pending
 
+- [\#487](https://github.com/arkworks-rs/algebra/pull/487) (`ark-poly`) Refactor `EvaluationDomain` trait for cosets:
```

Comment:
Let's move parts of these to "Breaking changes" and "Features" (there's usually nothing directly under `Pending`, only under the other subheadings.